ci/scripts/notify.py

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env python3
+
+import subprocess
+
+# Add new test keys here.
+# Add their corresponding owners (by slack username) here.
+# NOTE(kwannoel): we may have to migrate to use `slack_user_id`.
+# I use `slack_username` since it is more readable, but not officially supported in the docs.
+TEST_MAP = {
+    "test-notify": ["noelkwan", "noelkwan"],
```

Comment:
is this line for the default owner, i.e. if any test is not specified in the `TEST_MAP` but it fails, then it will notify `noelkwan`?

Reply:
Nope, it's just my internal test that the notification system work. If tests are not included here, there will be no notification for them at all.